blender/LilySurfaceScrapper/Scrappers/TexturesOneScrapper.py

```diff
@@ -0,0 +1,45 @@
+# Copyright notice?
+
+from .AbstractScrapper import AbstractScrapper
+from ..ScrappersManager import ScrappersManager
+
+class TexturesOneScrapper(AbstractScrapper):  
+    source_name = "Textures.one"
+    home_url = "https://www.textures.one"
+    
+    # There is probably something rotten about doing it this way, but I couldn't really figure out another way
```

Comment:
Those better be instance attributes (`self.foo`). You can define them in fetchVariantList because it is the first method of a scrapper instance called with the URL. To avoid duplicating the behavior of canHandleUrl, in particular the call to findSource, you can create a caching dictionnary that maps urls to the scrapper class and source URL. This cache will be a class attribute.

Reply:
I don't think that adding caching is a good idea. The execution of this part is already pretty fast, since it's just calling on each scrapper a function that most likely runs a simple if/e. If we added caching we would have to deal with edge cases like an URL being associated with a certain scrapper, but that scrapper isn't able to scrape all assets on that website.

If we were to add caching I'd suggest that it should be implemented in `ScrappersManager.py`, but I don't think it's worth it.